clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/cache/Keys.java

```diff
@@ -154,6 +175,10 @@ public static String getClusterKey(String account, String region, String cluster
     return buildKey(Namespace.ECS_CLUSTERS.ns, account, region, clusterName);
   }
 
+  public static String getApplicationKey(String name) {
```

Comment:
can we add a test for Application key formation to the `KeySpec`?

Reply:
updated